src/modules/mavlink/streams/MANUAL_CONTROL.hpp

```diff
@@ -68,10 +68,10 @@ class MavlinkStreamManualControl : public MavlinkStream
 			mavlink_manual_control_t msg{};
 
 			msg.target = mavlink_system.sysid;
-			msg.x = manual_control_setpoint.x * 1000;
-			msg.y = manual_control_setpoint.y * 1000;
-			msg.z = manual_control_setpoint.z * 1000;
-			msg.r = manual_control_setpoint.r * 1000;
+			msg.x = manual_control_setpoint.pitch * 1000;
+			msg.y = manual_control_setpoint.roll * 1000;
+			msg.z = manual_control_setpoint.throttle * 1000;
```

Comment:
@MaEtUgR Here, do we need to re-normalize it into [0, 1000] as well, like done in `mavlink_receiver.cpp`?

Reply:
Discussed: According to the spec (which itself is referring to the 'thrust vector component' direction, which shouldn't be considered in manual control signal itself: https://mavlink.io/en/messages/common.html#MANUAL_CONTROL, we are transparently sending back the actual throttle setting used, in the range that is specified in MAVLink [-1000, +1000], so it's fine.